### DISCUSSION TITLE
Save and load file

### DISCUSSION THREAD

**Discussion by KatNeko101** [skyline-emu/skyline#842] [Ideas]
How about add save and load file for the skyline emulator because if the game was restart sometimes and back to the prologue levels again that makes tired our hands or fingers and if was good idea or not

**Comment by rebornsunny2000** [comment]
If you mean save states, it is not possible for Switch emulation for a number of reasons already discussed in the discord server. Most Switch games have autosave features and if not, make a habit of saving often manually ingame.

**Reply by KatNeko101** [reply]
Oh I didn't not know was already discussed in discord server
